src/lib/y2network/config_reader/sysconfig.rb

```diff
@@ -32,13 +33,21 @@ class Sysconfig
       # @return [Y2Network::Config] Network configuration
       def config
         interfaces = find_interfaces
-        routing = Routing.new(tables: [load_routes])
+        routing = Routing.new(tables: [load_routes], forward_ipv4: forward_ipv4?, forward_ipv6: forward_ipv6?)
 
         Config.new(interfaces: interfaces, routing: routing, source: :sysconfig)
       end
 
     private
 
+      # sysctl keys, used as *single* SCR path components below
+      IPV4_SYSCTL = "net.ipv4.ip_forward".freeze
+      IPV6_SYSCTL = "net.ipv6.conf.all.forwarding".freeze
+      # SCR paths
+      SYSCTL_AGENT_PATH = ".etc.sysctl_conf".freeze
+      SYSCTL_IPV4_PATH = SYSCTL_AGENT_PATH + ".\"#{IPV4_SYSCTL}\""
+      SYSCTL_IPV6_PATH = SYSCTL_AGENT_PATH + ".\"#{IPV6_SYSCTL}\""
```

Comment:
why not having it already as Yast::Path? It allows even + operand https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/path.rb#L31

Reply:
just copy-pasted ... I'll rework it